README.md

```diff
@@ -20,6 +20,13 @@ Run `docl init` and supply the different configuration options based on your set
 * `--docker-host` should point to the docker endpoint.
 * `--source-root` should point to the root directory in which all cloudify related projects are cloned. This is used for mounting code
   from the host machine to the relevant manager directories.
+* (Optional) `[-u]|[--manager-image-url]` URL to the docker manager image. This must be set if you want to download a specific image when running `docl pull-image`.
+* (Optional) `[-a]|[--manager-image-commit-sha-url]` URL to the `sha1` checksum. This can save time if image already downloaded when 
+the `--manager-image-url` is set. 
+
+Using a dev branch:
```

Comment:
It's kinda of repeating it's self with the following, so you can remove it.

Add explanation on the docker flag